test/integration/lookout_controller_e2e_test.go

```diff
@@ -47,16 +47,25 @@ var _ = Describe("Armada Operator", func() {
 				Expect(err).NotTo(HaveOccurred())
 				Expect(secret.Data["lookout-e2e-1-config.yaml"]).NotTo(BeEmpty())
 
-				deployment := appsv1.Deployment{}
-				deploymentKey := kclient.ObjectKey{Namespace: "default", Name: "lookout-e2e-1"}
-				err = k8sClient.Get(ctx, deploymentKey, &deployment)
+				job := batchv1.Job{}
+				jobKey := kclient.ObjectKey{Namespace: "default", Name: "lookout-e2e-1-migration"}
+				err = k8sClient.Get(ctx, jobKey, &job)
 				Expect(err).NotTo(HaveOccurred())
-				Expect(deployment.Spec.Selector.MatchLabels["app"]).To(Equal("lookout-e2e-1"))
 
-				service := corev1.Service{}
-				serviceKey := kclient.ObjectKey{Namespace: "default", Name: "lookout-e2e-1"}
-				err = k8sClient.Get(ctx, serviceKey, &service)
-				Expect(err).NotTo(HaveOccurred())
+				// TODO: Migration Job never completes, so we can't test this yet
```

Comment:
A job will never complete in these tests.  The Job controller is not running in envtest.

Reply:
Good to know. Thanks!